CuratedContent/ManualWorkIsABug.md

```diff
@@ -0,0 +1,72 @@
+
+## Manual Work is a Bug
+
+<!-- deck text start -->
+Work to automate routine computing processes using an Agile incremental approach, starting by documenting the manual steps and then incrementally automating pieces of the process using scripts/code.
+Continuing to do routine processes repeatedly with manual execution is a bug!
+<!-- deck text end -->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe "Roscoe A. Bartlett")
+#### Publication date: May 27, 2024
+
+Resource information | Details
+:--- | :---
+Paper Title | Manual Work is a Bug
+Authors | Thomas A. Limoncelli
+Publication | [ACM Queue, March 14, 2018, Volume 16, Issue 1](https://queue.acm.org/detail.cfm?id=3197520)
+
+Nearly everyone who works with computers often has to perform routine tasks.
+Some people like system administrators (sysadmins) and DevOps engineers have **many** such tasks to perform.
+However, even other types of developers and users often have repetitive tasks that they need to execute repeatedly.
+And others in their teams and organizations often need to perform similar processes/tasks (or the same tasks, with minor variations).
+These routine processes may take long enough, be complex enough, have severe enough consequences if they are done in correctly, and/or be likely to be repeated enough to justify automating them to some degree.
+Thomas A. Limoncelli's article "Manual Work is a Bug" describes an Agile incremental process for developing increasing levels of automation for routine processes like these.
+(Thomas mainly speaks to sysadmins in this article, but the process and the principles described are the same for any individual or team that needs to perform routine tasks with computers.)
+
+The basic idea is to start by writing documentation right from the beginning while manually performing the steps for the first time for every process that is a potential candidate to become a routine (automatable) process.
```

Comment:
```suggestion
The basic idea is to start by writing documentation right from the beginning while manually performing the steps, for the first time, for every process that is a potential candidate to become a routine (automated) process.
```

Reply:
Applied in commit 34a11ccbd84db1ea44bcf3eaea9c31c584cfe4fa.